JamGame/Assets/Scripts/Utils/Raycast/RayCaster.cs

```diff
@@ -3,14 +3,14 @@
 using UnityEngine;
 using UnityEngine.EventSystems;
 
-namespace Common
+namespace Utils.Raycast
 {
-    public static class RayCastUtilities
+    public static class RayCaster
     {
         public static bool PointerIsOverUI(Vector2 screenPosition)
         {
             IEnumerable<GameObject> hitObjects = UIRayCast(ScreenPosToPointerData(screenPosition));
-            return hitObjects.Any(x => x.layer == LayerMask.NameToLayer("UI"));
+            return hitObjects.Any(x => x.GetComponentInParent<Blocker>() != null);
```

Comment:
This doesn't include case when blocker is *behind* target object. It's probably will not be a case but anyway will be nice to process it